_posts/2022-08-04-local-regional-working-group.md

```diff
@@ -0,0 +1,122 @@
+it can be helpful to learn about how other people use a particular tool
+like an IDE or which processes are particularly useful at another institution.
+Even when an organization does have a healthy network of RSEs, socializing
+with co-workers can have particular limitations. A local RSE group can provide
+an opportunity to socialize with other people that have a shared experience
+without the "co-worker" label. A local RSE group is a great way to stay
+connected to who is doing what in the area. It's an opportunity to create
+a local community of RSEs who support each other throughout their careers.
+
+## Resources
+
+The US-RSE Association is well-established and has many resources to help
+in advertising and coordinating local RSE groups. Since the US-RSE Community
+Building Workshop, the Local and Regional Working Group has met regularly
+and identified the following activities that enable establishing and
+leading a local group:
+
+### Formation of the Local and Regional Working Group
+
+The working group has been formed and is using the private Slack channel
+`#wg-local-regional` for organization. The working group meets monthly.
+If you are interested in joining the working group, reach out to the authors
+of this post via Slack.
+
+### Support for local groups by the members of the working group
+
+Some members of the working group will experiment by forming a local group
+in their own community. Since this is a new step for US-RSE, we intend
+to experiment ourselves and share our experience in a future post.
+Additionally, we will engage with anyone starting their own local
+group to offer resources and guidance as well as understand their
+own unique experience.
+
+### Regional group listings
+
+The general US-RSE Slack can be used to host a local-group Slack
+channel. By convention, these all are prefixed with `#rg-` followed by
+the name of the location; for example, `#rg-nola` for the New
+Orleans regional group. The following local and regional groups already
+exist in Slack. The membership counts are accurate as of the
+publication date of this post.
+
+| Location | Slack Channel | Membership |
+|---|---|---|
+| Arizona | `#arizona-rse` | 17 |
+| California | `#california-rse` | 38 |
+| Chicago Area, Illinois | `#rg-chicago-rse` | 8 |
+| Denver Area, Colorado (Front Range) | `#rg-frontrange` | 29 |
+| New England | `#rg-new-england` | 13 |
+| New Mexico | `#new-mexico-rse` | 9  |
+| New Orleans, LA (NOLA) | `#rg-nola` | 3  |
+| Pennsylvania | `#pennsylvania-rse` | 7  |
+
+Additionally, a layer has been added to the membership map
+showing the location of the active local groups. Each dot has an overlay
+that links to the corresponding Slack channel. This allows visitors
+to the map to quickly see if a local group exists in their area
+and communicate via Slack. New groups can be added by engaging 
+directly with this working group.
+
+![Regional Group Layer](/assets/posts/2022-08-04-local-regional-working-group.png)
```

Comment:
```suggestion
![Regional Group Layer]({{ site.baseurl }}/assets/posts/2022-08-04-local-regional-working-group.png)
```

Reply:
Locally this isn't an issue (I just confirmed), but it's not rendering correctly in the CIrcleCI preview, I _believe_ because of this. To be consistent, I think we should add this in here.